src/support/compare.ts

```diff
@@ -35,6 +35,53 @@ export interface DiffOptions {
 }
 
 /**
+ * Interface for a generic constructor function
+ */
+export interface Constructor {
+	new (...args: any[]): object;
+	prototype: object;
+}
+
+export interface ConstructDescriptor {
+	configurable?: boolean;
+	enumerable?: boolean;
+	writable?: boolean;
+}
+
+/**
+ * A record that describes a constructor function and arguments necessary to create an instance of
+ * an object
+ */
+export interface UnamedConstructRecord {
+	/**
+	 * Any arguments to pass the constructor function
```

Comment:
Any arguments to pass the constructor function -> Any arguments to pass to the constructor function ?